web/src/components/molecules/Request/Table/index.tsx

```diff
@@ -97,6 +110,12 @@ const RequestListTable: React.FC<Props> = ({
         }
         return <Badge color={color} text={request.state} />;
       },
+      filters: [
+        { text: "WAITING", value: "WAITING" },
+        { text: "APPROVED", value: "APPROVED" },
+        { text: "CLOSED", value: "CLOSED" },
+        { text: "DRAFT", value: "DRAFT" },
+      ],
```

Comment:
`t()` should be added here right?
```suggestion
      filters: [
        { text: t("WAITING"), value: "WAITING" },
        { text: t("APPROVED"), value: "APPROVED" },
        { text: t("CLOSED"), value: "CLOSED" },
        { text: t("DRAFT"), value: "DRAFT" },
      ],
```